DESCRIPTION

```diff
@@ -87,10 +87,12 @@ Suggests:
     dygraphs,
     tibble,
     fs,
-    callr (>= 2.0.0)
+    callr (>= 2.0.0),
+    rmarkdown
 SystemRequirements: pandoc (>= 1.12.3) - http://pandoc.org
-URL: https://rmarkdown.rstudio.com
+URL: https://rmarkdown.rstudio.com, https://github.com/rstudio/rmarkdown
```

Comment:
I made this change once https://github.com/rstudio/rmarkdown/pull/1230/files but decided to revert it later for reasons explained here: https://github.com/yihui/tinytex/pull/119.

Reply:
Looking back at my commit history, I did that for an important reason. I think it is important to be able to use the [changelog](https://rmarkdown.rstudio.com/docs/news/index.html) to link to the actual github issues. This for me is one of my biggest struggles with the current non-pkgdown repo is that I read the `NEWS.md` and cannot easily find the issues. If you don't provide a GitHub URL in this field, the issues in the [changelog](https://rmarkdown.rstudio.com/docs/news/index.html) look like this: 

<img width="995" alt="Screen Shot 2019-06-25 at 1 30 14 PM" src="https://user-images.githubusercontent.com/12160301/60131083-61c3ab00-974d-11e9-9be6-35866492ebb8.png">


Instead of:

<img width="995" alt="Screen Shot 2019-06-25 at 1 31 16 PM" src="https://user-images.githubusercontent.com/12160301/60131133-8750b480-974d-11e9-9a71-b044926422ad.png">

I actually think for usability this is one of the most important features of a `pkgdown` site, as it is much easier to stay abreast of the changes recorded in `NEWS.md` and trace back to the original issue/discussion.

Reply:
Yes, I understand that the links on issue numbers are important (which is why I always point users to Github releases like https://github.com/rstudio/rmarkdown/releases instead of `NEWS.md` in a repo). It will be nice if **pkgdown** can infer the Github links from `BugReports` field instead of `URL` in the package `DESCRIPTION`.

I'll use `https://github.com/rstudio/rmarkdown` as the sole value for `URL`.

Reply:
Sorry to step into your conversation but it's actually useful to include the pkgdown website URL in the `URL` field as well to enable cross-linking from other pkgdown websites, as explained in [this WIP vignette](https://github.com/r-lib/pkgdown/pull/1043/files#diff-6af677264efe6c1f5938e0c41b681767).